lib/administrate/search_term_parser.rb

```diff
@@ -0,0 +1,24 @@
+require "csv"
+
+module Administrate
+  class SearchTermParser
+    def self.parse(term)
+      all_terms = CSV.parse(term).first
```

Comment:
This usage of CSV is not explained. I guess it's done to ensure quotes and commas are parsed correctly. Perhaps add a spec to exemplify this? This way, if a surprised developer removes this in the future, they'll get a failure as an indication of what this is for.